numpy/_core/config.h.in

```diff
@@ -108,6 +108,8 @@
 #mesondefine HAVE_LDOUBLE_IBM_DOUBLE_DOUBLE_LE
 #mesondefine HAVE_LDOUBLE_IBM_DOUBLE_DOUBLE_BE
 
+#mesondefine HAVE_BLAS
```

Comment:
Style question (I really don't know if there is a real preference).  Should we use `NPY_` prefix for new additions?

Reply:
The `NPY` ones go in the public `numpyconfig.h` header. This is only exposed in the private `config.h` header (included by `npy_config.h`, another internal header).